scopes/ui-foundation/tree/file-tree/file-tree.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useMemo } from 'react';
+import { inflateToTree } from '@teambit/base-ui.graph.tree.inflate-paths';
+import { TreeContextProvider } from '@teambit/base-ui.graph.tree.tree-context';
+import { indentStyle } from '@teambit/base-ui.graph.tree.indent';
+import { RootNode } from '@teambit/base-ui.graph.tree.root-node';
+import { TreeNodeContext, TreeNodeRenderer } from '@teambit/base-ui.graph.tree.recursive-tree';
+
+type FileTreeProps = {
+  onSelect?: (id: string, event?: React.MouseEvent) => void;
+  selected?: string;
+  files: string[];
+  TreeNode?: any; //TreeNodeRenderer<PayloadType>; - what do we want here?
+};
+
+export function FileTree({ files, onSelect, selected, TreeNode }: FileTreeProps) {
+  const rootNode = useMemo(() => {
+    const tree = inflateToTree(files, (c) => c);
+    return tree;
```

Comment:
for dev/main labels, choose one of these solutions:

* calc in widget (node) => devFiles.has(node.id)
* attachPayload (map<Node, payload>() )
* files.map((filepath) => { id: filepath, isDev: true, ... }

Reply:
pass from code tree